tools/py/tests.py

```diff
@@ -58,7 +58,9 @@
     args = parser.parse_args()
     tests = args.tests
     print(test_folder[args.tests])
-    exit_code = pytest.main(["--tb=line", "-ra", str(test_folder[args.tests])])
+    exit_code = pytest.main(
+        ["--tb=line", "-ra", "-vv", "-s", str(test_folder[args.tests])]
```

Comment:
what are these new flags for ?

Reply:
to make sure that when the regtests fail, they will tell you why.